packages/server/src/components/LocationInput.svelte

```diff
@@ -0,0 +1,275 @@
+        </div>
+      </div>
+    </div>
+
+    <div class="field is-horizontal">
+      <div class="field-label is-small">
+        <label class="label" for="city">City</label>
+      </div>
+      <div class="field-body">
+        <div class="field">
+          <div class="control has-icons-right">
+            <input
+              class="input"
+              class:is-danger={$resourceForm.city.errors.length}
+                     autocomplete="address-level2"
+              type="text"
+              placeholder="City"
+              bind:value={resource.address.city} />
+            {#if $resourceForm.city.errors.length}
+              <span class="icon is-small is-right">
+                <i class="fas fa-exclamation-triangle" />
+              </span>
+            {/if}
+          </div>
+          {#if $resourceForm.city.errors.includes('required')}
+            <p class="help is-danger">An address is required</p>
+          {/if}
+        </div>
+      </div>
+    </div>
+
+    <div class="field is-horizontal">
+      <div class="field-label is-small">
+        <label class="label" for="state">State</label>
+      </div>
+      <div class="field-body">
+        <div class="field">
+          <div class="control has-icons-right">
+            <input
+              class="input"
+              class:is-danger={$resourceForm.state.errors.length}
+                     autocomplete="address-level1"
+              type="text"
+              placeholder="State"
+              bind:value={resource.address.state} />
+            {#if $resourceForm.state.errors.length}
+              <span class="icon is-small is-right">
+                <i class="fas fa-exclamation-triangle" />
+              </span>
+            {/if}
+          </div>
+          {#if $resourceForm.state.errors.includes('required')}
+            <p class="help is-danger">A state is required</p>
+          {/if}
+        </div>
+      </div>
+    </div>
+
+    <div class="field is-horizontal">
+      <div class="field-label is-small">
+        <label class="label" for="zip">ZIP</label>
+      </div>
+      <div class="field-body">
+        <div class="field">
+          <div class="control has-icons-right">
+            <input
+              class="input"
+              class:is-danger={$resourceForm.zip.errors.length}
+                     autocomplete="postal-code"
+              type="text"
+              placeholder="ZIP"
+              bind:value={resource.address.zip} />
+            {#if $resourceForm.zip.errors.length}
+              <span class="icon is-small is-right">
+                <i class="fas fa-exclamation-triangle" />
+              </span>
+            {/if}
+          </div>
+          {#if $resourceForm.zip.errors.includes('required')}
+            <p class="help is-danger">A ZIP code is required</p>
+          {/if}
+        </div>
+      </div>
+    </div>
+  </div>
+
+  {#if mapShowing}
+    <div class=" { placeSelected ? 'column' : 'hidden' } ">
```

Comment:
I'm good with always showing the map so we can avoid the awkward columns

https://user-images.githubusercontent.com/6598084/108302248-d2101300-7160-11eb-82f7-0105168d6058.mp4

Reply:
Cool cool I'll change the map to be always showing.
I was playing around with hiding the map mostly because if someone doesn't use any of the Google suggested results (i.e. they enter the whole address manually), then the map simply sits there without updating, which I feel could be somewhat confusing to people? I guess a different fix to that might be to kind of reverse the process and update the map if they enter everything manually 🤔

Reply:
Oh ya I get that for sure 👍 I could see a user thinking that it can be interacted with, but we can save that for the user testing being put together for the provider portal.